website/client/components/chat/chatMessages.vue

```diff
@@ -202,8 +202,17 @@ export default {
 
       if (!profile._id) {
         const result = await this.$store.dispatch('members:fetchMember', { memberId });
-        this.cachedProfileData[memberId] = result.data.data;
-        profile = result.data.data;
+        if (!result.data) {
```

Comment:
There might be other reasons the data doesn't respond. Can we check this for a 404? I think result.code will have the http code.